src/query/interpreter.cpp

```diff
@@ -1164,14 +1213,14 @@ PreparedQuery PrepareDumpQuery(ParsedQuery parsed_query, std::map<std::string, T
 }
 
 PreparedQuery PrepareIndexQuery(ParsedQuery parsed_query, bool in_explicit_transaction,
-                                std::map<std::string, TypedValue> *summary, InterpreterContext *interpreter_context,
-                                utils::MemoryResource *execution_memory) {
+                                std::vector<Notification> *notifications, InterpreterContext *interpreter_context,
+                                utils::MemoryResource * /*execution_memory*/) {
```

Comment:
I think you can remove these not used parameters, as non of the functions has to follow a strict interface.

Reply:
Same for `PrepareConstraintQuery`.